### ISSUE TITLE
[Request- Requires Fork] decreasing decimals for main assets that are 100% owned by a single address

### ISSUE THREAD

**Issue by jeroz1** [RavenProject/Ravencoin#443]
I know the stance of the current developers but I see **many** requests on our social media to make this possible because users did not realize that they had to change the default setting. I therefore extend their voice to this requested change. 

It could be implemented as a stealth feature for assets that unlocks the decimals if the requirements are met. 

Edit: I am also aware that this needs a fork and, if deemed feasible, will take until the next phase to be implemented.

**Comment by JackAsparrow** [comment]
> #431 fixes this.

No. We're referring to having the ability to reduce decimals on assets that have already been created with 8 decimals but haven't been transferred or had sub-assets created yet.

**Comment by shakeshaker** [comment]
JackAsparrow is right. Not sure #431 fixes this.

**Comment by jeroz1** [comment]
#431 does not fix this. It merely changes the default setting to 0. @russkidooski

**Comment by blondfrogs** [comment]
@jeroz1 I completely understand why this would be useful. I will look into this and see if it is possible.

**Comment by jeroz1** [comment]
@blondfrogs I figured as much, thanks :)

**Comment by shakeshaker** [comment]
@blondfrogs Let's hope this will be implemented in the future. Too bad the default was 8 decimals. I suspect thousands of asset were issued this way where the owner doesn't want 8 decimals. I think it's a big one.

**Comment by gazingjar** [comment]
We badly need this. Not only this, but letting delete subassets assets that are still set to 'redistributable.' It's a long shot, but it would be hugely beneficial.

**Comment by THATDONFC** [comment]
One more vote for this. It just makes sense!

**Comment by RikkiRATTOE** [comment]
I vote for this change as well as an ability to lower the total supply of a token where none of the supply has ever been distributed.

**Comment by wehard** [comment]
Yes, this would be a nice feature.

**Comment by entegral** [comment]
thanks for looking into this @blondfrogs, this feature would be great

**Comment by liqdmetal** [comment]
this feature need to be added! bump. there are planned forks already, so let's get this included asap so it can  go on testnet.

**Comment by blondfrogs** [comment]
So, a little bit more information about this feature. In order for this change to occur, ever node must have every address and the asset balance in that address. This was removed as it wasn't needed by the software to increase speeds and performance. 
This means at this time, that there is no way to change the decimal to a smaller number. I will continue to investigate ways on how to do this type of change but for now this isn't a priority of the development team.

**Comment by cfox** [comment]
This is super specific and I don't know when it will be a priority.  Take a hack at it!

**Comment by vincent-raven** [comment]
Still think this should be looked at before we get too far past it...
